src/harpstrata/IsActive/getActiveIdsPair/getCounterpartActiveIds/index.test.ts

```diff
@@ -0,0 +1,33 @@
+import type { IsActiveProps } from '../../types'
+import { PitchIds, C, D, E, F } from '../../../Pitch'
+import { DegreeIds, ROOT, SECOND, THIRD, FOURTH } from '../../../Degree'
+
+import { getCounterpartDegreeIds, getCounterpartPitchIds } from './index'
+
+const degreeMatrix = [
+  [ ROOT , SECOND ],
+  [ THIRD, FOURTH ],
+]
+const pitchMatrix = [
+  [ C, D ],
+  [ E, F ],
+]
+const baseIsActiveProps: IsActiveProps = {
+  degreeMatrix, pitchMatrix, activeIds: []
+}
+
+test('getCounterpartDegreeIds returns the DegreeIds[] for a given PitchIds[]', () => {
```

Comment:
DegreeIds[] and PitchIds[] should be renamed to ActiveDegreeIds and ActivePitchIds